lombok.config

```diff
@@ -0,0 +1,3 @@
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
I think we can remove this file from the repo because it gets generated automatically now